tests/cli/superset/export_test.py

```diff
@@ -509,6 +526,52 @@ def test_export_users(mocker: MockerFixture, fs: FakeFilesystem) -> None:
     ]
 
 
+def test_export_users_force_unix_eol_enable(
+    mocker: MockerFixture,
+    fs: FakeFilesystem,
+) -> None:
+    """
+    Test the ``export_users`` command.
```

Comment:
Could we add to the docstring that the test is with `force_unix_eol` set to `True`? Something like:
```
Test the ``export_users`` command with ``--force-unix-eol`` flag.
```